code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -403,7 +403,17 @@
 
 	return acting_object.drop_location()
 
+/obj/item/device/electronic_assembly/calc
+	icon_state = "setup_small_calc"
```

Comment:
The names are all the same? This would break savefiles.

Reply:
Oh, shit, yeah. I'll fix that while I'm giving the default sprites a facelift